recipes/cyrus-sasl/all/conanfile.py

```diff
@@ -52,7 +52,7 @@ class CyrusSaslConan(ConanFile):
         "with_scram": True,
         "with_otp": True,
         "with_krb4": True,
-        "with_gssapi": False, # FIXME: should be True
+        "with_gssapi": True,
```

Comment:
Well, looking at krb5 recipe it's a bad idea to enable this option by default. krb5 recipe raises for too many common settings (Windows, Macos, and also when openssl is static). Look at v1 pipeline, nothing is built.